app/resonant-laboratory/ansible/library/girder.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/python
```

Comment:
If you wanted, you could use the galaxy role for no other purpose than to bring this module in automatically.